demos/projects/ESPRESSIF/adu/README.md

```diff
@@ -35,6 +35,11 @@ This sample will allow you to update an ESP32 over the air (OTA) using Azure Dev
     For other Operating Systems or to update an existing installation, follow [Espressif official documentation](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/get-started/index.html#get-started).
 
 1. [Powershell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell?view=powershell-7.2)
+
+1. Azure CLI and Azure IoT Module
+
+See steps to install both [here](https://learn.microsoft.com/en-us/azure/iot-hub-device-update/create-update?source=recommendations#prerequisites).
```

Comment:
```suggestion
See steps to install both [here](https://learn.microsoft.com/azure/iot-hub-device-update/create-update?source=recommendations#prerequisites).
```
Relevant to the other files and the arduino PR as well

Reply:
Good fix! Thanks.